tensorflow_addons/layers/normalizations_test.py

```diff
@@ -246,6 +246,95 @@ def test_groupnorm_conv(self):
             model.fit(x=x, y=y, epochs=1)
             self.assertTrue(hasattr(model.layers[0], "gamma"))
 
+    def test_groupnorm_correctness_1d(self):
+        model = tf.keras.models.Sequential()
+        norm = GroupNormalization(input_shape=(10,), groups=2)
+        model.add(norm)
+        model.compile(loss="mse", optimizer="rmsprop")
+
+        x = np.random.normal(loc=5.0, scale=10.0, size=(1000, 10))
+        model.fit(x, x, epochs=5, verbose=0)
+        out = model.predict(x)
+        out -= tf.keras.backend.eval(norm.beta)
+        out /= tf.keras.backend.eval(norm.gamma)
+
+        np.testing.assert_allclose(out.mean(), 0.0, atol=1e-1)
+        np.testing.assert_allclose(out.std(), 1.0, atol=1e-1)
+
+    def test_groupnorm_different_groups(self):
+        norm1 = GroupNormalization(axis=1, groups=2, input_shape=(10, 6))
+        norm2 = GroupNormalization(axis=1, groups=1, input_shape=(10, 6))
+        norm3 = GroupNormalization(axis=1, groups=10, input_shape=(10, 6))
```

Comment:
Rather than repeating code for each value of `group`, you can use parameterized tests from absl. [Example usage](https://github.com/tensorflow/addons/blob/49637c3bc8c53b11a2a6490048ef824cf4c46b44/tensorflow_addons/layers/wrappers_test.py#L87).

Reply:
Thanks @Squadrick.